core/src/test/scala/matryoshka/spec.scala

```diff
@@ -288,8 +291,8 @@ class MatryoshkaSpecs extends Specification with ScalaCheck with ScalazMatchers
       case _ => t.map(_.head).embed
     }
 
-  "Recursive" should {
-    "isLeaf" should {
+  "Recursive" >> {
+    "isLeaf" >> {
```

Comment:
I am not convinced you can nest `should` in `specs2` `3.x`.

I would be inclined to go with this just because they are unit tests and it does not read very well in all cases. But I can try and see if I am imagining things about not being able to nest `should` if you prefer.